addons/account/static/src/components/mail_attachments/mail_attachments.js

```diff
@@ -33,6 +33,18 @@ export class MailAttachments extends Component {
         return file.name.replace(/^.*\./, "");
     }
 
+    isImage(file) {
+        return [
+            "image/jpg",
+            "image/jpeg",
+            "image/jpe",
+            "image/png",
+            "image/svg+xml",
+            "image/gif",
+            "image/webp"
+        ].includes(file.mimetype);
```

Comment:
Where did you get the list of those mimetypes? I know it is for testing but from my research I stumbled upon a list that differs from yours.
If you know for a fact we can accept all types of images you could simplify to
```suggestion
        return file.mimetype.startsWith("image/");
```